README.md

```diff
@@ -23,9 +23,11 @@ Then you need to wire up the FaencyProvider which will hold the context with the
 ```jsx
 import { FaencyProvider } from '@traefiklabs/faency';
 
-const App = () => <FaencyProvider primaryColor="$blue8">{/* your app */}</FaencyProvider>;
+const App = () => <FaencyProvider primaryColor="blue">{/* your app */}</FaencyProvider>;
```

Comment:
I think this is outdated, as `FaencyProvider` ignores `primaryColor` prop:
https://github.com/traefik/faency/blob/master/components/FaencyProvider.tsx#L89


I see no story or unit test where we set a primary color.

Am I missing something?